src/components/Registration/BusinessLookup.vue

```diff
@@ -0,0 +1,264 @@
+              <span>{{businessName}}</span>
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col cols="12" sm="3">
+              <label>Incorporation Number</label>
+            </v-col>
+            <v-col cols="12" sm="9">
+              <span>{{identifier}}</span>
+            </v-col>
+          </v-row>
+          <v-row no-gutters>
+            <v-col cols="12" sm="3">
+              <label>Business Number</label>
+            </v-col>
+            <v-col cols="12" sm="9">
+              <span>{{businessNumber}}</span>
+            </v-col>
+          </v-row>
+        </v-col>
+        <v-col cols="3">
+          <v-btn v-if="!hasBusinessLookupChanges" text color="primary" id="bl-change-btn" @click="onChangeClicked()">
+            <v-icon small>mdi-pencil</v-icon>
+            <span>Change</span>
+          </v-btn>
+
+          <div v-else id="bl-more-actions">
+            <v-btn text color="primary" id="bl-undo-btn" @click="emitUndo()">
+              <v-icon small>mdi-undo</v-icon>
+              <span>Undo</span>
+            </v-btn>
+            <v-menu offset-y left nudge-bottom="4" v-model="dropdown">
+              <template v-slot:activator="{ on }">
+                <v-btn text small color="primary" id="bl-menu-btn" v-on="on">
+                  <v-icon>{{dropdown ? 'mdi-menu-up' : 'mdi-menu-down'}}</v-icon>
+                </v-btn>
+              </template>
+              <v-btn text color="primary" id="bl-more-changes-btn" class="py-5"
+                @click="onChangeClicked(); dropdown = false">
+                <v-icon small color="primary">mdi-pencil</v-icon>Change</v-btn>
+            </v-menu>
+          </div>
+        </v-col>
+      </v-row>
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
+import { Debounce } from 'vue-debounce-decorator'
+import { BusinessLookupIF, BusinessLookupResultIF } from '@/interfaces'
+enum States {
+  INITIAL = 'initial',
+  SEARCHING = 'searching',
+  SHOW_RESULTS = 'show results',
+  NO_RESULTS = 'no results',
+  SUMMARY = 'summary'
+}
+@Component({})
+export default class BusinessLookup extends Vue {
+  /** Whether to show errors. */
+  @Prop({ required: true })
+  readonly showErrors!: boolean
+  /** The BusinessLookup object. */
+  @Prop({ required: true })
+  readonly businessLookup!: BusinessLookupIF
+  /** Class for BusinessLookup services. */
+  @Prop({ required: true })
+  readonly BusinessLookupServices!: any
+  /** Whether to display Change features. */
+  @Prop({ default: false })
+  readonly hasBusinessLookupChanges!: boolean
+  // enum for template
+  readonly States = States
+  // local variables
+  private state = States.INITIAL
+
+  private searchField = ''
+  private searchResults: Array<BusinessLookupResultIF> = []
+  private selectedBusiness = null
+
+  /** V-model for dropdown menu. */
+  private dropdown: boolean = null
```

Comment:
You probably copied this from other code, but why null? Does it need to be not-false and not-true? (I know that, in some cases, it needs to be null. I'm just wondering about here.)

Reply:
Make sense. Will look into it in the next ticket.